data/MitAbhay.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "Abhay Kumar Mittal",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open Source | Devops | Full Stack Developer",
+  "avatar": "https://avatars.githubusercontent.com/u/76659113?v=4",
```

This link should be used instead as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/MitAbhay.png",
```